scripts/mainnet-val-setup.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
+set -e
+
+command_exists () {
+    type "$1" &> /dev/null ;
+}
+
+if command_exists go ; then
+    echo "Golang is already installed"
+else
+  echo "Install dependencies"
+  sudo apt update
+  sudo apt install build-essential jq wget git -y
+
+  wget https://dl.google.com/go/go1.15.2.linux-amd64.tar.gz
+  tar -xvf go1.15.2.linux-amd64.tar.gz
+  sudo mv go /usr/local
+
+  echo "" >> ~/.bashrc
+  echo 'export GOPATH=$HOME/go' >> ~/.bashrc
+  echo 'export GOROOT=/usr/local/go' >> ~/.bashrc
+  echo 'export GOBIN=$GOPATH/bin' >> ~/.bashrc
+  echo 'export PATH=$PATH:/usr/local/go/bin:$GOBIN' >> ~/.bashrc
+  
+  clear
+fi
+
+source ~/.bashrc
+
+echo "CAUTION!"
+echo "-- If Regen was previously installed, the following step will remove ~/.regen from your system. Are you sure you would like to continue?--"
+
+select yn in "Yes" "No"; do
+    case $yn in
+        Yes ) rm -rf ~/.regen; break;;
+        No ) exit;;
+    esac
+done
+
+DAEMON=regen
+DENOM=uregen
+CHAIN_ID=regen-1
+PERSISTENT_PEERS="69975e7afdf731a165e40449fcffc75167a084fc@104.131.169.70:26656,d35d652b6cb3bf7d6cb8d4bd7c036ea03e7be2ab@116.203.182.185:26656,ffacd3202ded6945fed12fa4fd715b1874985b8c@3.98.38.91:26656"
+
+echo "install regen-ledger"
+git clone https://github.com/regen-network/regen-ledger $GOPATH/src/github.com/regen-network/regen-ledger
```

Comment:
This requires `sudo` but also not necessary to clone into the `src` directory and then to change into that directory. Might be easier to just clone the repo into the current home directory.